### ISSUE TITLE
Only 3 guides showing up in the guides list?

### ISSUE THREAD

**Issue by Elijah-Bodden** [hackclub/blot#503]
![image](https://github.com/hackclub/blot/assets/106613755/93d0a6b1-2af0-48dd-ad68-fb6411d7886f)
For some reason only 3 of the many guides in `guides` show up on the website (at least on arc)

**Comment by BrightTheBackpack** [comment]
the guides that show up are controled by the metadata file in /guides/ directory. I think this is intentional

**Comment by leomcelroy** [comment]
Yes the other guides are a bit out of date. It would be nice to update them. I wanted to have a somewhat coherent style to the guides we had on release too.
